third_party/icu/shim/common/unicode/uidna.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
```

Comment:
@danzh2010 @alyssawilk do you know how reasonable it will be to add proper Chromium URL support for eliding unicode translation? I think this shim is fine for now, but ideally we can move away from needing to workarounds like this.

Reply:
I'm not sure, but cc @davidschinazi @ianswett just to have on our list.

Reply:
It's definitely doable to pull in the full Chromium IDN logic, but that has significant impact on binary size - not sure what the full set of design goals and requirements is here though

Reply:
@DavidSchinazi I think probably what we need is a way to build GURL without Unicode support (probably via defining build flag etc). Is that something viable?

Reply:
@dio I think that should be doable by using IDN from the OS:
https://source.chromium.org/chromium/chromium/src/+/master:url/BUILD.gn;l=72;drc=a82b58b53bbc8b7e843c0624a64a4428ad2fa7bd